### PR TITLE
6.2 update

### PR DESCRIPTION
**Pull request by pyrodie18**
##### SUMMARY
Updated roles to support Zabbix 6.2 referenece #761 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
Various Roles

##### ADDITIONAL INFORMATION
N/A

### CODE REVIEW

**Comment by pyrodie18** [comment]
@D3DeFi , @BGmot , and anyone else.  I can't figure out why this thing keeps failing the sanity checks.  Mind taking a peak?

**Comment by pyrodie18** [comment]
Fixes #764   and #761

**Comment by D3DeFi** [comment]
@pyrodie18 
```
ERROR: Found 7 pylint issue(s) which need to be resolved:
ERROR: plugins/httpapi/jsonrpc.py:95:4: arguments-renamed: Parameter 'data' has been renamed to 'request_method' in overridden 'HttpApi.send_request' method
ERROR: plugins/module_utils/wrappers.py:31:123: used-before-assignment: Using variable 'ZBX_IMP_ERR' before assignment
ERROR: plugins/modules/zabbix_action.py:555:43: format-string-without-interpolation: Using formatting for a string that does not have any interpolated variables
ERROR: plugins/modules/zabbix_group.py:143:119: used-before-assignment: Using variable 'ZBX_IMP_ERR' before assignment
ERROR: plugins/modules/zabbix_map.py:764:117: used-before-assignment: Using variable 'PYDOT_IMP_ERR' before assignment
ERROR: plugins/modules/zabbix_map.py:766:117: used-before-assignment: Using variable 'WEBCOLORS_IMP_ERR' before assignment
ERROR: plugins/modules/zabbix_map.py:768:111: used-before-assignment: Using variable 'PIL_IMP_ERR' before assignment@
```

**Comment by D3DeFi** [comment]
please have a look at #775 , after merging, you can rebase this branch and tests should pass

File: molecule/zabbix_server/molecule copy.yml
Comment (D3DeFi) [inline molecule/zabbix_server/molecule copy.yml:1]: Is this by any chance an accidental file? Name would indicate so

**Review by D3DeFi — APPROVED** [review]
Overall good patch, thank you!

Sanity tests should stop failing after rebasing on top of #775 

Please check my comment, if the _molecule/zabbix_server/moluecule copy.yml_ is indeed a duplicate.

File: molecule/zabbix_server/molecule copy.yml
Comment (pyrodie18) [reply molecule/zabbix_server/molecule copy.yml:1]: Ya it is.  I modified molecule because I haven't been able to get workflows to work locally and forgot to trash the copy.  I'll fix it and rebase tonight.  Any idea why sanity would just now start throwing errors since I didn't touch any of the plugins.

Comment (D3DeFi) [reply molecule/zabbix_server/molecule copy.yml:1]: since we are testing against `develop` branch from ansible/ansible as well, we are subject to such surprises when new standards are merged in. This sometimes happens even before it gets announced at ansible-collections/new-for-maintainers, not much to do.

So it was just a bad luck

**Comment by pyrodie18** [comment]
Looks like that did the trick @D3DeFi ....thanks

**Comment by dj-wasabi** [comment]
Nice! 👍

**Comment by D3DeFi** [comment]
This looks like a merge to me rather than rebase. It should be fine in the end after it is merged, but is very confusing to review as it includes changes from the already merged PRs.

What I usually recommend is:
```bash
git remote add upstream <this repo>
git fetch -a upstream
git checkout main
git rebase upstream/main
git checkout <this branch>
git rebase main
git push origin main
git push origin <this branch>
```

**Comment by D3DeFi** [comment]
@BGmot @mu1f407 can you give this PR a review too? Once approved we can merge. Ignore `include_pattern` logic and changes to `plugins/modules/*` as those were already merged to main

**Comment by BGmot** [comment]
Do I understand it right - we test only the latest Zabbix version for each distro? For example - only 6.2 for Debian 11 despite we claim that we support Zabbix 6.2, 6.0, 5.0, 4.0 ?

**Comment by pyrodie18** [comment]
> This looks like a merge to me rather than rebase. It should be fine in the end after it is merged, but is very confusing to review as it includes changes from the already merged PRs.
> 
> What I usually recommend is:
> 
> ```shell
> git remote add upstream <this repo>
> git fetch -a upstream
> git checkout main
> git rebase upstream/main
> git checkout <this branch>
> git rebase main
> git push origin main
> git push origin <this branch>
> ```

I ran rebase but apparently screwed it up.  Sorry about that

**Comment by pyrodie18** [comment]
> Do I understand it right - we test only the latest Zabbix version for each distro? For example - only 6.2 for Debian 11 despite we claim that we support Zabbix 6.2, 6.0, 5.0, 4.0 ?

Currently yes, I mentioned this in #766 .  One of the things I would like to do as part of 2.0 is update the matrix to include all versions on all OSs that we support.  But we may want to start a separate discussion about that since it's going to increase the number of tests that we run.

**Comment by dj-wasabi** [comment]
> But we may want to start a separate discussion about that since it's going to increase the number of tests that we run.

That is indeed why I always used to test latest. But mostly also as I think someone has to pay for all the tests that we do, don't wanted to do something that would increase the costs that much..

**Comment by D3DeFi** [comment]
> > But we may want to start a separate discussion about that since it's going to increase the number of tests that we run.
> 
> That is indeed why I always used to test latest. But mostly also as I think someone has to pay for all the tests that we do, don't wanted to do something that would increase the costs that much..

As an open source, I think we have some credits for Github Actions on the repo for free. I think its their free 2000CI/CD minutes or something

**Review by mu1f407 — APPROVED** [review]
Looks good! Thank you.

**Comment by pyrodie18** [comment]
> > > But we may want to start a separate discussion about that since it's going to increase the number of tests that we run.
> > 
> > 
> > That is indeed why I always used to test latest. But mostly also as I think someone has to pay for all the tests that we do, don't wanted to do something that would increase the costs that much..
> 
> As an open source, I think we have some credits for Github Actions on the repo for free. I think its their free 2000CI/CD minutes or something

Created a dedicated topic for this on #777

**Comment by D3DeFi** [comment]
Thanks all for discussion and @pyrodie18 for the heavy lifting here! :)

**Pull request merged**
